keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -0,0 +1,493 @@
+|                              |InodesUsed          |Container        |CRI                   |CRI                        |                                                |
+|                              |                    |                 |                      |                           |container_fs_io_current                         |
+|                              |                    |                 |                      |                           |container_fs_io_time_seconds_total              |
+|                              |                    |                 |                      |                           |container_fs_io_time_weighted_seconds_total     |
+|                              |                    |                 |                      |                           |container_fs_read_seconds_total                 |
+|                              |                    |                 |                      |                           |container_fs_reads_bytes_total                  |
+|                              |                    |                 |                      |                           |container_fs_reads_merged_total                 |
+|                              |                    |                 |                      |                           |container_fs_reads_total                        |
+|                              |                    |                 |                      |                           |container_fs_sector_reads_total                 |
+|                              |                    |                 |                      |                           |container_fs_sector_writes_total                |
+|                              |                    |                 |                      |                           |container_fs_write_seconds_total                |
+|                              |                    |                 |                      |                           |container_fs_writes_bytes_total                 |
+|                              |                    |                 |                      |                           |container_fs_writes_merged_total                |
+|                              |                    |                 |                      |                           |container_fs_writes_total                       |
+|                              |                    |                 |                      |                           |                                                |
+|UserDefinedMetrics            |All Fields          |Container        |cAdvisor              |CRI or N/A                 |                                                |
+|No Equivalent in Stats Summary|                    |                 |                      |                           |container_processes                             |
```

Comment:
There is pod-level process stats in the summary API: https://github.com/kubernetes/kubernetes/blob/97d40890d00acf721ecabb8c9a6fec3b3234b74b/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go#L95

Reply:
ah, good catch, fixed!